lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -240,6 +240,7 @@ export class PublicClientApplication extends ClientApplication implements IPubli
                 return this.acquireTokenByRefreshToken(silentRequest, requestWithCLP).catch((refreshTokenError: AuthError) => {
                     const isServerError = refreshTokenError instanceof ServerError;
                     const isInteractionRequiredError = refreshTokenError instanceof InteractionRequiredAuthError;
+                    const attemptSsoRTNotFound = (refreshTokenError.errorCode === BrowserConstants.NO_TOKENS_FOUND_ERROR) && this.config.auth.attemptSSO;
```

Comment:
What do we think about just attempting ssoSilent when `No_tokens_found` is thrown and let the cache lookup policy determine whether or not ssoSilent should be skipped? Not a huge fan of introducing another config option...

Reply:
I am fine with it. I was conflicted too, just trying to make sure we do not break anyone who assumes this. Do you mean something like this:

```javascript

if(noTokensFoundError) {
    acquireTokenByIframe(...);
 }
 ```
 
 or 
 
 ```javascript

if(noTokensFoundError || CachePolicy.Skip) {
    ssoSilent(...);
 }
 ```
 
 Option 1, we can accommodate with the above change by removing the config option altogether. It will attempt SSO for anyone who has default cache look up policy.

Reply:
Updated to not use a `config` param. This behavior will be default (try SSO if RT is missing in the cache) unless specified otherwise in cache policy or `forceRefresh`